bitnami/discourse/values.yaml

```diff
@@ -312,34 +316,46 @@ persistence:
   ##   GKE, AWS & OpenStack)
   ##
   # storageClass: "-"
-  ##
+
```

Comment:
We usually tend to keep common blocks together like this, and space out the different sections to ease the reading. For example, take a look:

https://github.com/bitnami/charts/blob/ccdebcbc99403900d0248fa22adc17c450c5261a/bitnami/airflow/values.yaml#L237-L248

https://github.com/bitnami/charts/blob/ccdebcbc99403900d0248fa22adc17c450c5261a/bitnami/airflow/values.yaml#L312-L322